.gitignore

```diff
@@ -16,6 +16,15 @@
 #    Multiple authors (IBM Corp.) - initial implementation and documentation
 ###############################################################################
```

Comment:
general comment: modified files should have their "rightmost" copyright date updated to the current year